src/Makefile

```diff
@@ -2,6 +2,6 @@ CC=g++
 CFLAGS=-Wall -std=c++11
 
 all:
-	$(CC) $(CFLAGS) main.cpp FGame.cpp FCharacter.cpp FEngine.cpp FLevel.cpp FObject.cpp FVector.cpp FWeapon.cpp FProjectile.cpp FPowerup.cpp -lsfml-graphics -lsfml-window -lsfml-system
+	$(CC) $(CFLAGS) main.cpp FGame.cpp FCharacter.cpp FEngine.cpp FLevel.cpp FObject.cpp FVector.cpp FWeapon.cpp FProjectile.cpp FPowerup.cpp FTile.cpp -lsfml-graphics -lsfml-window -lsfml-system
 clean:
 	rm -f *.o FightR2
```

Comment:
Yeah we will need a better makefile. Yours looks nice, but I also think we need (in the future anyway) more options than just `make all`, because I rarely need to recompile *all* files every time.